_templates/exchanges.html

```diff
@@ -168,6 +168,12 @@ <h3 id="mexico"><img src="/img/flags/MX.png" alt="Mexican flag">Mexico</h3>
   </div>
   <div>
     <div>
+      <h3 id="nepal"><img src="/img/flags/NP.png" alt="Nepalese flag">Nepal</h3>
+      <p>
+        <a href="https://www.bitsewa.com/">Bitsewa</a><br>
```

Comment:
@tuladhar Please remove the `<br>` since there is only one entry for Nepal at the moment.